src/storage/FirebaseStorageDirective.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+  "use strict";
+
+  function FirebaseStorageDirective($firebaseStorage, firebase) {
+    return {
+      restrict: 'A',
+      priority: 99, // run after the attributes are interpolated
+      scope: {},
+      link: function (scope, element, attrs) {
+        // $observe is like $watch but it waits for interpolation 
+        // Ex: <img firebase-src="{{ myUrl }}"/> 
+        attrs.$observe('firebaseSrc', function (newVal) {
+          if (newVal !== '' && newVal !== null && newVal !== undefined) {
```

Comment:
`newVal` is triggered once interpolation is done. I believe once interpolation is done they are equal, I went with `attrs.firebaseSrc` for assignment because it read more clearly. But I can make the variable name of `newVal` more explicit.